src/components/offers/details/SingleMenu.js

```diff
@@ -1,9 +1,8 @@
-import React from 'react';
+import React, {useState} from 'react';
```

Comment:
To ma prawo działać? XDD

Reply:
a czemu nie? XDDDD Kuba ma tak w każdym swoim PRze XDDD ale tak, jest to legitny sposób na import rzeczy w react XDDD

Reply:
Ja tylko robię co pan WebStorm każe, a pan WebStorm się bardzo denerwuje kiedy ja `use JSX` bez `importing React`